app/models/viewing_party.rb

```diff
@@ -0,0 +1,7 @@
+class ViewingParty < ApplicationRecord
+  has_many :users, through: :user_viewing_parties
+  has_many :user_viewing_parties
```

Comment:
in regards to your comment in your spec, the only thing that I can see here is that these lines need to be swapped (2 and 3) because you need to call user viewing parties before users. I don't know if that'll fix your issue though. Worth a shot trying?

Reply:
great call. Didn't realize the order on this mattered!